src/main/webapp/resources/js/pages/projects/share/ShareSamples.jsx

```diff
@@ -10,22 +11,26 @@ import { updatedLocked, updateMoveSamples } from "./shareSlice";
  * @returns {JSX.Element}
  * @constructor
  */
-export function ShareSamples({ samples = [], redirect }) {
+export function ShareSamples({ samples = [] }) {
   const dispatch = useDispatch();
-  const { originalSamples, locked, remove } = useSelector(
-    (state) => state.shareReducer
-  );
+  const { associated, originalSamples, locked, remove, currentProject } =
+    useSelector((state) => state.shareReducer);
 
   const SHOW_SAMPLES = samples.length > 0;
   const SHOW_NO_SAMPLES_WARNING = samples.length === 0;
   const SHOW_SOME_SAMPLES_WARNING =
     SHOW_SAMPLES && samples.length < originalSamples.length;
+  const SHOW_ASSOCIATED = SHOW_SAMPLES && associated.length > 0;
 
   return (
-    <>
+    <Space direction="vertical" style={{ width: `100%` }}>
+      <Typography.Title level={5}>
+        {i18n("ShareSamplesList.title")}
+      </Typography.Title>
       {SHOW_SAMPLES && (
-        <Space direction="vertical" style={{ width: `100%` }}>
-          <SharedSamplesList list={samples} />
+        <>
+          <ShareAssociated />
```

Comment:
Move this outside the SHOW_SAMPLES logic and add it below the title where it checks for SHOW_ASSOCIATED

Reply:
Thanks!